InternxtDesktop/Services/BackupsService.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  BackupsService.swift
+//  InternxtDesktop
+//
+//  Created by Richard Ascanio on 1/8/24.
+//
+
+import Foundation
+import RealmSwift
+
+class BackupsService: ObservableObject {
+    @Published var foldernames: [FoldernameToBackup] = []
+    @Published var urls: [URL] = []
+    @Published var isBackupButtonEnabled: Bool = false
```

Comment:
Don't save UI related stuff here, instead, define if the button is enabled in a view, as a sideffect of something else, for example `var isBackupsButtonEnabled = folderNames.count > 0`